src/main/scala/chiseltest/internal/Utils.scala

```diff
@@ -0,0 +1,21 @@
+package chiseltest.internal
+
+import os.Path
+
+private[chiseltest] object Utils {
```

Comment:
Since this seems to only be used in one place, I would prefer if you could just define it as a private function in whatever class is using it.

Reply:
placed function where used. I'd like to note this code is super useful if you are trying to see or test a giant ugly call to verilator or some other program